src/supermarket/app/views/profile/edit.html.erb

```diff
@@ -55,7 +52,6 @@
 
       <% if ROLLOUT.active?(:github) %>
         <div class="content" id="github-accounts">
-          <p>You must have at least one linked GitHub account in order for us to match your contributions on GitHub with your signed Contributors License Agreement.</p>
```

Comment:
With this text removed, the GitHub account tab now appears as:

![screen shot 2016-12-06 at 1 13 34 pm](https://cloud.githubusercontent.com/assets/517302/20937647/d2f211ac-bbb5-11e6-8aa8-455647596ef6.png)

The button seems so lonely. And I'm not sure why I would want to link my GitHub account. Instead of removing text  entirely, what do you think about replacing it with something along the following?

> Linking your account lets other cookbook shoppers find you on GitHub for feedback, collaboration, and kudos. The link requests access only to your GitHub account's public information.